views/partials/tips.hbs

```diff
@@ -1,4 +1,175 @@
+  </div>
+</div>
+<div id="tips-container" class="section-container">
+    <div class="tips-wrapper">
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_1@2x.png" alt="Account Security Tip #1">
+                <span class="tip-numeral medium">1</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Use a different password<br>for every account</h3>
+              </div>
+              <p>You can't prevent a data breach, but you can limit your exposure by always using different passwords for different websites.</p>
+          </div><!--closes tip-->
+      </div><!--closes tip-container 1-->
+
+      <!-- tip # 2 -->
+
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_2@2x.png" alt="Account Security Tip #2">
+                <span class="tip-numeral medium">2</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Create strong<br>passwords</h3>
+              </div>
+              <p>Hackers try to steal passwords by using lists of common passwords and by guessing. The longer and more random your password is, the harder it will be to steal.</p>
+          </div><!--closes tip-->
+      </div><!--closes tip-container 2-->
+
+      <!-- tip #3 -->
+
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_3@2x.png" alt="Account Security Tip #3">
+                <span class="tip-numeral medium">3</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Make strong<br>security questions</h3>
+              </div>
+              <p>Websites don't care if your security questions are accurate - they only care that your answers match every time. So give answers to the security questions that are long and random, like your passwords.</p>
+          </div><!--closes tip-->
+      </div><!--closes tip-container 3-->
+
+      <!-- tip # 4 -->
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_4@2x.png" alt="Account Security Tip #4">
+                <span class="tip-numeral medium">4</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Use a password<br>manager</h3>
+              </div>
+              <p>Password manager like 1Password, LastPass, or Dashlane can generate strong passwords for you, remember them for you, and fill them into websites so you don't have to type them in.</p>
```

Comment:
Should we link the password managers to their respective sites?

Reply:
@pdolanjski can tell us for sure, but I say no - don't link to the external sites. These are good products, but we shouldn't link off to external sites unless/until we've done some more thorough review of their products. A link could be interpreted as an endorsement.

Reply:
Mentioning them is an endorsement/recommendation here. The link is just a courtesy at this point.
But yeah, I guess they could just right click the name and select the "Search Google for 1Password" text, then click in the Google search results to get to the site. A 2-step detour (versus just `<a href="https://1password.com" target="_blank" rel="nofollow noopener noreferrer">...</a>` &mdash; or whatever our magic incantation would be).

Reply:
I think there's a problem with the copy here - it should be "Password manager**s**" (plural), no?